src/services/api/index.ts

```
@@ -1,19 +1,20 @@
 const api = axios.create({
   baseURL: process.env.NEXT_PUBLIC_PODCAST_API,
-})
+});
 
 api.interceptors.response.use((response: AxiosResponse) => {
   if (response.config.url.match(/\/podcasts/)) {
-    response.data = response.data instanceof Array
-      ? response.data.map(transformPodcast)
-      : transformPodcast(response.data)
+    response.data =
+      response.data instanceof Array
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)